docs/topics/i18n/translation.txt

```diff
@@ -567,6 +567,13 @@ As with all template tags, this tag needs to be loaded in all templates which
 use translations, even those templates that extend from other templates which
 have already loaded the ``i18n`` tag.
 
+.. note::
+
+    Internationalized strings will not be escaped when rendered in a template.
+    This allows you to include HTML in translations, for example for emphasis,
```

Comment:
Should we put comma before `for emphasis`? :thinking:

Reply:
Not sure, feels weird to me, but I'm not a native speaker.

Reply:
As you know me neither :man_shrugging:  :smile: